website/docs/advanced/targeting.md

```diff
@@ -194,6 +194,67 @@ The exact value that will be served to the users that fall into that fraction. D
 | Whole Number   | Integer      | any whole number within the range of `Int32`      |
 | Decimal Number | Double       | any decimal number within the range of `double` |
 
+## Stickiness & Consistency
+
+The percentage-based targeting is sticky by design and consistent across all SDKs.
+
+Percentage-based targeting is based on the identifier passed in the `User Object` to get `getValue()` methods in the SDKs. 
```

Comment:
A get szónak nincsen értelme a getValue() előtt, szerintem elírás.
Javaslat: Percentage-based targeting is implemented in the SDKs. The percentage is based on the identifier which is passed to the `getValue()` method through the `User Object`.

Reply:
Fixed